dashboards/autorelay/autorelay.json

```diff
@@ -255,10 +259,22 @@
           "legendFormat": "current reservations",
           "range": true,
           "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "libp2p_autorelay_desired_reservations",
+          "hide": true,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "B"
```

Comment:
@marten-seemann it's a bit hacky but it does work. We set the desired reservations on relay_finder start.

Reply:
fixed this.